src/types/requests/HttpByIDRequest.ts

```diff
@@ -3,3 +3,7 @@ import { HttpDatabaseProjectRequest } from './HttpDatabaseRequest';
 export default interface HttpByIDRequest extends HttpDatabaseProjectRequest {
   ID: string;
 }
+
+export interface HttpRestByIDRequest extends HttpDatabaseProjectRequest {
+  id: string;
```

Comment:
Should use HttpByIDRequest only I think.

Reply:
According to my comment above, I'm using this interface as a temporary interface that will replace the old uppercased HttpByIDRequest when every endpoints will be refactored

Reply:
Done, rolled back IDs in query.